jail-supybot.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: configure supybot
+  hosts: supybot
+  gather_facts: no
+  connection: local
+  sudo: yes
+  vars:
+    env_name: venv
+    web_host_name: "supybot.buildbot.net"
+    master_service_name: "supybot"
+
+  roles:
+  - base
+  - role: user
+    user_id: "{{ worker_account }}"
+    user_name: Buildbot Worker Account
+  - role: virtualenv
+    venv_user: "{{ worker_account }}"
+    venv_home_dir: "{{ getent_passwd[worker_account].4 }}"
```

Comment:
Discovered recently the `expanduser` filter in ansible. It will expand paths with tildes in them. This includes the form "~user". I don't think we have a variable for this (maybe `worker_home` being ~bb-infra), but I think it would be a comprehension improvement over the `getent_paswd`.

Since it appears in multiple places, might be useful to have a separate PR for that.